src/engine/qcommon/parse.cpp

```diff
@@ -2971,8 +2971,8 @@ static int Parse_Directive_include( source_t *source )
 		{
 			// buffer too small?
 			path[ MAX_QPATH - 1 ] = 0;
-			strncpy( path, source->includepath, MAX_QPATH - 1 );
```

Comment:
I don't think you should subtract 1 from the buffer size with the new functions. It doesn't really matter though, since on the 0.52.0 branch I deleted this file (and moved it into the cgame).